tests/integration/iop-integration-test-defaults.yaml

```diff
@@ -21,6 +24,10 @@ spec:
 
     gateways:
       istio-ingressgateway:
+        meshExpansionPorts:
```

Comment:
isn't this the default already?

Reply:
Yes, but there is an issue with Istioctl manifest apply and how it merges multiple yaml files (which is what is used in test integration framework), see here https://github.com/istio/istio/issues/24377.  If one yaml file in the list doesn't have it `meshExpansionPorts` declared on a list, it breaks `--set`

Reply:
removed with the addition of RemoteClusterValues